src/http/api/thread.unit.test.ts

```diff
@@ -133,6 +135,7 @@ describe('Thread API', () => {
                                 Audience.Public,
                                 'Test Post 789 (reply to Test Post 123)',
                                 'Test Post 789 Excerpt',
+                                null,
```

Comment:
Out of scope for this PR given - but wonder if we should be using named parameters to improve readability / not having to insert params in the same exact order everywhere?